streaming/base/dataset.py

```diff
@@ -922,14 +922,16 @@ def resample_streams(
         sample_ids = np.concatenate(sample_ids).astype(np.int64)
         return shuffle_units, sample_ids
 
-    def _share_work(self, sample_ids: NDArray[np.int64]) -> Tuple[SharedMemory, SharedMemory]:
+    def _share_work(self,
+                    sample_ids: NDArray[np.int64]) -> Tuple[SharedMemory, Optional[SharedMemory]]:
```

Comment:
```suggestion
                    sample_ids: NDArray[np.int64],) -> Tuple[SharedMemory, Optional[SharedMemory]]:
```
cleaner formatting with trailing ,

Reply:
done